pkg/cmd/chat/channel/channel.go

```diff
@@ -0,0 +1,230 @@
+package channel
+
+import (
+	"encoding/json"
+	"errors"
+	"text/tabwriter"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/cheynewallace/tabby"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getCmd(),
+		createCmd(),
+		deleteCmd(),
+		updateCmd(),
+		listCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-channel --type [channel-type] --id [channel-id]",
+		Short: "Returns a channel",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetStreamClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			chanType, _ := cmd.Flags().GetString("type")
+			chanId, _ := cmd.Flags().GetString("id")
+
+			r, err := c.Channel(chanType, chanId).Query(cmd.Context(), &stream.QueryRequest{})
+			if err != nil {
+				return err
+			}
+
+			w := tabwriter.NewWriter(cmd.OutOrStdout(), 0, 0, 2, ' ', 0)
+			t := tabby.NewCustom(w)
+			t.AddHeader("CID", "Member count", "Created By", "Last Message At", "Created At", "Updated At", "Custom Data")
+			t.AddLine(r.Channel.CID,
+				r.Channel.MemberCount,
+				r.Channel.CreatedBy.ID,
+				r.Channel.LastMessageAt.Format(time.RFC822),
+				r.Channel.CreatedAt.Format(time.RFC822),
+				r.Channel.UpdatedAt.Format(time.RFC822),
+				r.Channel.ExtraData)
+			t.Print()
```

Comment:
it looks like we need a generic way of printing stuff. This is something good to extract into its own task

Reply:
✅